tests/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ include_directories(
 add_executable(tst_utils tst_utils.cpp)
 target_link_libraries(tst_utils
     ayatana-common
-
+    ${GLIB_LIBRARIES}
```

Comment:
Why commit this and the test file where it is required separately?

Reply:
We loose context with these atomics commits.

Normally, I'd expect: code change, unit tests and build system adaptations in one commit. With some proper commit message that describes what this change does.